modules/gapi/cmake/standalone.cmake

```diff
@@ -15,6 +15,8 @@ file(GLOB FLUID_includes "${FLUID_ROOT}/include/opencv2/*.hpp"
                          "${FLUID_ROOT}/include/opencv2/gapi/own/*.hpp"
                          "${FLUID_ROOT}/include/opencv2/gapi/fluid/*.hpp")
 file(GLOB FLUID_sources  "${FLUID_ROOT}/src/api/g*.cpp"
+                         "${FLUID_ROOT}/src/api/kernels_core.cpp"
+                         "${FLUID_ROOT}/src/api/operators.cpp"
```

Comment:
It bloats our standalone build .a file. Can you please measure the size before/after?

Also -- not sure if `kernels_core` will work in the standalone mode at all - since the dispatch thing is different when it is used in the IE preproc.